src/validators/actix.rs

```diff
@@ -155,19 +158,21 @@ where
 		let svc = self.service.clone();
 
 		// We want to skip running the validator if we are not able to find a matching path from the listed valid paths provided by the user
-		match self.routes.clone() {
+		match &self.routes {
 			Some(route_matches) => {
 				// If the user only wants to apply authentication to a select amount of routes, we handle that logic here
 				let path = req.path();
 				// Check if the path was NOT contained inside of the routes specified by the user...
-				if !route_matches.contains(&path.to_owned()) {
```

Comment:
Unfortunatelly `slice::contains` not stellar-API can't let us search with a `&str`, so  changed to `Iterator::find`

Reply:
Great catch!